GFS_layer/GFS_typedefs.F90

```diff
@@ -885,6 +885,7 @@ module GFS_typedefs
     character(len=32)    :: iau_forcing_var(20)  ! list of tracers with IAU forcing
     real(kind=kind_phys) :: iaufhrs(7)      ! forecast hours associated with increment files
     logical :: iau_filter_increments, iau_drymassfixer
+    logical :: override_surface_radiative_fluxes  ! Whether to use Statein to override the surface radiative fluxes
```

Comment:
With the Coupling type now holding the data, I think the comment needs to be updated.